common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/WebViewAuthorizationFragment.java

```diff
@@ -264,6 +266,16 @@ public boolean onTouch(final View view, final MotionEvent event) {
         mWebView.setWebViewClient(webViewClient);
     }
 
+    /**
+     * NOTE: In the case that AADWebViewClient is using EoClientCertAuthChallengeHandler to handle CBA,
```

Comment:
This seems more like a comment than a JavaDoc. I think this is specifically for line 276

Reply:
Made change to regular comment.